app/models/application_record.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
   include Warren::BroadcastMessages
+  extend Squishify # Provides squishify
```

Comment:
Don't think the comment is very helpful 😆 
Maybe could be `for minifying JavaScript and CSS` ... if that is correct?